pipelines/pingcap/tidb/latest/merged_integration_prisma_test.groovy

```diff
@@ -0,0 +1,160 @@
+// REF: https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-pipeline
+// Keep small than 400 lines: https://issues.jenkins.io/browse/JENKINS-37984
+// should triggerd for master and latest release branches
+@Library('tipipeline') _
+
+final K8S_NAMESPACE = "jenkins-tidb"
+final GIT_CREDENTIALS_ID = 'github-sre-bot-ssh'
+final POD_TEMPLATE_FILE = 'pipelines/pingcap/tidb/latest/pod-merged_integration_prisma_test.yaml'
+final REFS = readJSON(text: params.JOB_SPEC).refs
+
+pipeline {
+    agent {
+        kubernetes {
+            namespace K8S_NAMESPACE
+            yamlFile POD_TEMPLATE_FILE
+            defaultContainer 'nodejs'
```

Comment:
use this default container seems that it may cause the compilation of tidb-server binary  step to fail.  Because the current tidb mater branch compilation requires go version 1.20.

Reply:
It is recommended to set up a separate golang container on file `pod-merged_integration_prisma_test.yaml`